documentation/modules/configuring/proc-listing-connector-offsets.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-listing-connector-offsets-{context}']
+= Listing connector offsets
+
+[role="_abstract"]
+To track connector offsets using `KafkaConnector` resources, add the `listOffsets` configuration. 
```

Comment:
```suggestion
To track connector offsets using `KafkaConnector` resources, add the `listOffsets` configuration and annotate the resource. 
```

Reply:
we mention annotating the resource at the end of the paragraph